tests/utils/common.sh

```diff
@@ -0,0 +1,31 @@
+# Exit on any error
+set -e
+# Helper(s)
+
+check_db_bench_workload_completion() {
+  WORKLOAD=$1
+  DBB_OUT=$2
+  if [ $(grep -wc -E "$WORKLOAD\s+:" $DBB_OUT) -ne 1 ]; then
+    echo "$(tput setaf 1)ERROR: the $WORKLOAD did not complete$(tput sgr 0)" 1>&2
+    return -1
+  fi
+  return 0
+}
+
+utest_run_fillrandom_dbbench_workload() {
```

Comment:
Since this has a single caller, there is no point having it in common.sh, move it in to the caller script in stead, this just makes the code harder to read, same for the others

Reply:
This is a common.sh specific for the utils directory and will have specific functions for these tests.  
This can be used in another script in future if needed, so i thought of keeping it here. I think it will make sense to keep it here for any future needs.

Reply:
no, only put things in common.sh if they are common

Reply:
Put things in common when they are used in more than one place